### PR TITLE
Updated doc section with glide

### LINKED ISSUE #116: No glide, glide-vc documentation
There is no documentation about adding new dependencies to the project. glide related docs will help.

### PR DESCRIPTION
**Pull request by surajnarwade**
Resolves #116,
Added development.md file which spefices instructions related to development
with opencompose as well as `glide` and `glide-vc` documentation for vendor update.
(referred from kompose development guide)

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+1. Go to https://github.com/redhat-developer/opencompose
+2. Click the "Fork" button (at the top right)
+
+### Clone your fork
+
+The commands below require that you have $GOPATH. We highly recommended you put OpenCompose code into your $GOPATH.
+
+```console
+git clone https://github.com/$YOUR_GITHUB_USERNAME/opencompose.git $GOPATH/src/github.com/redhat-developer/opencompose
```
Comment (concaf) [inline docs/development.md]: Let's remove `.git` from `https://github.com/$YOUR_GITHUB_USERNAME/opencompose.git`

File: docs/development.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+### Committing changes to your fork
+
+```console
+git commit
+git push -f origin myfeature
+```
+
+### Creating a pull request
+
+1. Visit https://github.com/$YOUR_GITHUB_USERNAME/opencompose.git
```
Comment (concaf) [inline docs/development.md]: Let's lose the `.git`

File: docs/development.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+### Using glide
+
+#### Adding new dependency
+1. Update `glide.yml` file.
+
+  Add new packages or subpackages to `glide.yml` depending if you added whole new package as dependency or
+  just new subpackage.
+
+2. Run `glide update --strip-vendor` to get new dependencies.
+   Than run `glide-vc --only-code --no-tests` to delete all unnecessary files from vendor.
```
Comment (concaf) [inline docs/development.md]: s/Than/Then

Comment (concaf) [reply docs/development.md]: @kadel our Makefile contains `glide-vc --only-code --no-tests --no-test-imports --no-legal-files`, should be put that here instead?

**Review by concaf — CHANGES_REQUESTED** [review]

File: docs/development.md (hunk 1 above)
Comment (surajnarwade) [reply docs/development.md]: @containscafeine , any specific reason for this ?

Comment (concaf) [reply docs/development.md]: Well, it's not required even though it works. The general idea is the we directly copy from the URL in the browser and do a `git clone`, and the URL does not have a `.git`.

**Comment by surajnarwade** [comment]
@containscafeine , I have updated with doc with new `make` commands for `glide` and `glide-vc`

**Comment by cdrage** [comment]
LGTM!

**Pull request merged**